apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/grpc/ToProtoValueCodecs.java

```diff
@@ -21,6 +22,17 @@
 import java.util.UUID;
 
 public class ToProtoValueCodecs {
+  /**
+   * Ok, this formatter is combination of info from:
+   *
+   * <ul>
+   *   <li>https://stackoverflow.com/questions/43360852/cannot-parse-string-in-iso-8601-format-lacking-colon-in-offset-to-java-8-date
+   *   <li>https://stackoverflow.com/questions/34637626/java-datetimeformatter-for-time-zone-with-an-optional-colon-separator
+   * </ul>
+   */
+  private static final DateTimeFormatter ISO_OFFSET_DATE_TIME_OPTIONAL_COLON =
```

Comment:
would be nice to have in the javadocs the formats that would be supported by this.. I guess `[]` means optional, and while `[.SSS]` is clear, the two different options using the `X` is confusing for me?

Reply:
Ok I can add it. As I understand it, `X` variants are for with-colon (`08:00`) as well as without (`0800`). unfortunately even `DateTimeFormat` Javadocs are bit confusing on what exactly is allowed; SO answer gives bit more practical answer.